src/services/ApiConfigStore.ts

```diff
@@ -0,0 +1,28 @@
+// our own mini redux store for storing api configs :))
+
+import { ConfigurationParameters, Configuration } from './api';
+
+class ApiConfigStoreClass {
+  private configParams: ConfigurationParameters = {
+    // TODO: load this from a config instead
+    basePath:
+      process.env.NODE_ENV === 'development'
+        ? 'http://localhost:3000'
+        : 'http://dev.api.hknucsd.com',
```

Comment:
Shouldn't this be for dev branch instead then if basePath is dev.api.hknucsd.com?

Reply:
these urls shouldn't even be here and should be loaded from process.env instead - just threw something together for demo